src/Build/Xml/ProjectXmlUtilities.cs

```diff
@@ -42,6 +42,21 @@ internal static void VerifyThrowProjectNoChildElements(XmlElementWithLocation el
             }
         }
 
+        /// <summary>
+        /// Throw an invalid project exception if there are any child elements with a different name than expected
+        /// </summary>
+        internal static void VerifyThrowProjectOnlyAllowedChildrenWithName(XmlElementWithLocation element, string allowedChildName)
```

Comment:
I'm a little confused by this. All children need to have a single name? I thought there were two you cared about? Maybe add a test?

Reply:
Yeah basically there was a test that verified that the node had no children in case there wasn't a TaskFactory specified. Now I'm relaxing the test to verify that it only has a child of a given name (Parameters). Since we still want to disallow the actual body, but we do want to allow Parameters.